canonical-web-and-design/ubuntu.com
# Build /data/kafka page

## Done

- Build `/data/kafka` page
- Build  contact form linked from the page
- Update redirect 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/data/kafka or https://ubuntu-com-11697.demos.haus/data/kafka
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare page against [copy doc](https://docs.google.com/document/u/0/d/1n6T7kbXECxd_bx2u8M5-bulLrCdySvDo2WcsEYmVRkE/edit)

## Issue / Card

Fixes [#5360](https://github.com/canonical-web-and-design/web-squad/issues/5360) and [#5361](https://github.com/canonical-web-and-design/web-squad/issues/5361)
